src/app/features/teacher/pages/teacher-home/teacher-home.component.html

```diff
@@ -0,0 +1,68 @@
+<section class="section">
+  <header class="section__header">
+    <h2 class="section__title">Plano de desenvolvimento individual</h2>
+    <p>
+      Olá, professor(a).
+      <br />
+      Vamos juntos acompanhar a evolução dos nossos estudantes? Nosso objetivo é
+      evoluir as habilidades humanas dos nossos estudantes.
+    </p>
+  </header>
+
+  <button mat-flat-button class="section__button" (click)="openDialog()">
+    Ficou com alguma duvida? <strong>Clique aqui</strong>
+  </button>
+
+  <div class="filters">
```

Comment:
`filters` não está claro que pertence ao bloco section. Sugiro trocar para  `section__filters` para manter a hierarquia

Reply:
Beleza vou alterar, mas na convenção BEM eu poderia ter blocos dentro de blocos? A minha intenção era que `filters` fosse um novo bloco dentro de section.

Reply:
Na convenção BEM, você pode sim ter blocos dentro de blocos, mas depende do contexto:

Se filters é independente: Trate como um novo bloco, algo como .filters. Isso faz sentido se ele for reutilizável ou puder existir fora de section.

Se filters depende de section: Ele deve ser um elemento, como .section__filters, porque faz parte do bloco section.

A ideia é sempre separar o que é independente (bloco) do que pertence a algo maior (elemento).

Reply:
Ah, agora entendi, faz sentido separar então.